applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+'require form';
+'require fs';
+'require view';
+
+return view.extend({
+	load: function() {
+		return fs.list('/dev').then(function(devs) {
+			return devs.filter(function(dev) {
+				return dev.name.match(/^[sh]d[a-z]$/);
+			});
+		});
+	},
+
+	render: function(devs) {
+		var m, s, o;
+		m = new form.Map('hd-idle', _('HDD Idle'), _('HDD Idle is a utility program for spinning-down external disks after a period of idle time.'));
+
+		s = m.section(form.TypedSection, 'hd-idle', _('Settings'));
+		s.anonymous = true;
+		s.addremove = true;
+		s.addbtntitle = _('Add new hdd setting...');
+
+		o = s.option(form.Flag, 'enabled', _('Enable'));
+		o.rmempty = false;
+
+		o = s.option(form.Value, 'disk', _('Disk'));
+		devs.forEach(function(dev) {
+			o.value(dev.name);
+		});
+
+		o = s.option(form.Value, 'idle_time_interval', _('Idle time'));
+		o.default = 10;
+
+		o = s.option(form.ListValue, 'idle_time_unit', _('Idle time unit'));
+		o.value('seconds', _('s'));
```

Comment:
Since these translation labels are very short and ambiguous, could you add a translation context by passing a second argument? Something like `_('s', 'Abbreviation for seconds')` - the context will not be visible to the user but it'll be part of the `*.po` files to give translators a hint on what `s` is supposed to mean.